<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>621</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_LineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: none;
border-bottom: 3px solid rgb(241, 92, 34);
background-color: Transparent;
color:black;
padding: 5px 5px 5px;
}
QLineEdit:Focus{
border: 2px solid rgb(241, 92, 34);
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Password_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PASSWORD</string>
    </property>
   </widget>
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>USERNAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Login_Button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>318</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: none;
background-color: rgb(28, 130, 255);
border-radius: 15px;
color: black;
}
QPushButton:Hover{
border:none;
background-color:rgb(23,130, 255);
border-radius:15px;
color:red;
}
QPushButton:Pressed {
border:5px solid rgb(28, 130, 255);
background-color: rgb(28, 140, 255);
border-radius: 15px;
color: red;
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="schName_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nanchang University</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_LineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: none;
border-bottom: 3px solid rgb(241, 92, 34);
background-color: Transparent;
color:black;
padding: 5px 5px 5px;
}
QLineEdit:Focus{
border: 2px solid rgb(241, 92, 34);
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Characters Are Not Accepted (e.g: ?/&gt;,.';)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
